docs/root/configuration/overview/extension.rst

```diff
@@ -80,8 +80,9 @@ for HTTP filters.
 
 Extension config discovery service has a :ref:`statistics
 <subscription_statistics>` tree rooted at
-*<stat_prefix>.extension_config_discovery.<extension_config_name>*. In addition
-to the common subscription statistics, it also provides the following:
+*<stat_prefix>.extension_config_discovery.<extension_config_name>*. For HTTP
```

Comment:
If we are changing the stat tree anyway, did you consider actually inverting this and making this something like:

`extension_config_discovery.<stat_prefix>.<extension_config_name>` ?

It seems like this would be more clear as we use ECDS in more places? WDYT?

/wait-any

Reply:
Yeah, I agree this is more consistent with listener manager and cluster manager stats. Updated.